src/pages/study-rooms/components/create-study-room-modal.tsx

```diff
@@ -1,43 +1,23 @@
-import { useState } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { Container, TextField } from '@mui/material';
-import { createStudyRoom } from '@/apis/study-room/create-study-room';
-import { ROUTE_PATH } from '@/constant/routes';
+import { TextField } from '@mui/material';
 import { MODAL_KEYS } from '@/constant/modal';
-import { setLocalStorage } from '@/utils/storage';
 import useModal from '@/hooks/useModal';
+import useCreateStudyRoomForm from '../hooks/useCreateStudyRoomForm';
 import Modal from '@/components/common/modal/modal';
+import CreateStudyRoomSelects from './create-study-room-select-fields';
```

Comment:
컴포넌트 이름을 그대로 import해오면 좋을 것 같네요~
```suggestion
import CreateStudyRoomSelectFields from './create-study-room-select-fields';
```

Reply:
근데 해당 컴포넌트 내에서 표현되는 UI를 보니까 지금 작성된 이름보다 조금 더 명확하게 작성해볼만 한 것 같아요~ StepTime을 선택하는 필드들을 모아놓은 것 같아서요!
실제로 select 자체가 스터디룸을 생성하는 역할(지금은 생성하는데 필요한 부품 정도?)을 하는건 아니기도 하구요~